andela/hc-rogue-one
# Ft notify often #156577843

#### What does this PR do?
Notify user when job is run too often
#### Description of Task to be completed?
User should receive a notification when a job is run too often
The job/check should get an icon indicating the job is running too often
#### How should this be manually tested?
1. Install the project and install the requirements.txt
2.Set up email configurations from an email server, `host`, `user`, `password`, `port`
3:Change the email backend to support the smtp server you are going to use
4: Create a job/check
5: Run the job `too often` by pinging the check URL multiple times
6: You should receive a an email informing you of the same and the job should get an icon indicating the job has run too often.

#### What are the relevant pivotal tracker stories?
#156577843

## Checklist: I completed these to help reviewers




  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`

- [] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`
